src/Components/Admins/Form/index.js

```diff
@@ -102,6 +131,19 @@ const Form = () => {
 
   return (
     <div className={styles.container}>
+      <Modal
+        showModal={showModal}
+        closeModal={closeModal}
+        actionEntity={submit}
+        titleText="Save"
+        spanObjectArray={[
+          {
+            span: 'are you sure you want to save these changes?'
```

Comment:
I'd rather starting messages with a capital letter
```suggestion
            span: 'Are you sure you want to save these changes?'
```